docs/fetching_credentials/README.md

```diff
@@ -0,0 +1,60 @@
+# Fetching Credentials
+
+As seen [here](https://support.demisto.com/hc/en-us/articles/115002567894), it is possible to integrate with 3rd party credentials 
+vaults for Demisto to use when authenticating with integrations. This article provides an example of such integration.
+
+## Requirements
+
+In order to fetch credentials to the Demisto credentials store, the integration needs to be able to retrieve credential objects 
```

Comment:
In order to fetch credentials from the Demisto credentials store, the integration needs to be able to retrieve credential objects

Reply:
Actually it's to - you're fetching from the vault to Demisto

Reply:
Ah okay then you're good :)